src/Workspaces/Remote/ServiceHub/Services/ServiceHubServiceBase.cs

```diff
@@ -60,10 +61,10 @@ protected RoslynServices RoslynServices
             }
         }
 
-        protected Task<Solution> GetSolutionAsync()
+        protected Task<Solution> GetSolutionAsync(OptionSet options = null)
```

Comment:
Can we make things non-optional?  It makes it super simple for someone to accidently do the wrong thing.  We shoudl probably document all the spots that pass null and explain why that's the right thing to do.